nix/openssl_1_0_2_fips.nix

```diff
@@ -0,0 +1,42 @@
+{ pkgs }:
+pkgs.stdenv.mkDerivation rec {
+  pname = "openssl";
+  version = "1.0.2-fips";
+
+  # TODO: build OpenSSL 1.0.2-fips
```

Comment:
Not sure I follow what the TODO here is. Can you open an issue and link to it explaining a bit more what's missing?

Reply:
This shouldn't be in this MVP; I failed to remove it. Basically building openssl 1.0.2 fips requires you download 2 tarballs. Doing that is, I think, possible in nix -- but not easily.

In this issue here:

https://github.com/aws/s2n-tls/issues/3841 #5 here